core/embed/prodtest/main.c

```diff
@@ -91,6 +105,15 @@ static void vcp_readline(char *buf, size_t len) {
   }
 }
 
+static void vcp_printf_ex(const char *fmt, ...) {
```

Comment:
`_ex` suggests to me an extended argument version, but this seems to be "no newline" version
maybe rename original to `vcp_println` and this one to `vcp_print`?

Reply:
e14114794714d5f8982a2c765fbe99d8d72c159e and e14114794714d5f8982a2c765fbe99d8d72c159e